### PR TITLE
chore: clean-up packages and bump versions

### PR DESCRIPTION
**Pull request by GabrielChouinardLetourneau**
GitHub Issue: #

## PR Type

What kind of change does this PR introduce?

- Other... Please describe:
Chore clean up of packages that weren't supposed to be in the Client head and bump to newest versions for Chinook.

## PR Checklist 
Please check if your PR fulfills the following requirements:

- [ ] Interface members are XML documented
- [ ] Documentation (XML or comments) has been added and/or existing documentation has been updated
- [ ] [Architecture documents](./doc/Architecture.md) have been updated
- [ ] Tested on all relevant platforms

### CODE REVIEW

File: src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj [hunk 1]
```diff
@@ -27,13 +27,13 @@
...
-    <PackageReference Include="Chinook.SectionsNavigation.Uno.WinUI" Version="1.1.0" />
-    <PackageReference Include="Chinook.DataLoader.Uno.WinUI" Version="1.1.0" />
+    <PackageReference Include="Chinook.DynamicMvvm.Uno.WinUI" Version="1.1.2" />
+    <PackageReference Include="Chinook.BackButtonManager.Uno.WinUI" Version="1.1.2" />
+    <PackageReference Include="Chinook.SectionsNavigation.Uno.WinUI" Version="1.1.2" />
+    <PackageReference Include="Chinook.DataLoader.Uno.WinUI" Version="1.1.1" />
     <PackageReference Include="ExtendedSplashScreen.Uno.WinUI" Version="0.3.0-dev.38" />
-    <PackageReference Include="Nventive.Persistence.Uno.WinUI" Version="0.2.0-dev.26" />
-    <PackageReference Include="Nventive.View.Uno.WinUI" Version="0.4.1-feature.dotnet6.53" />
+    <PackageReference Include="Nventive.Persistence.Uno.WinUI" Version="0.2.0-dev.28" />
```
Comment (carlh98) [inline src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj:35]: Use [0.3.0-dev.30](https://www.nuget.org/packages/Nventive.Persistence.Uno.WinUI/0.3.0-dev.30)

File: src/app/ApplicationTemplate.Presentation/Framework/Startup/IExtendedSplashscreenController.cs [hunk 2]
```diff
@@ -17,7 +17,7 @@ public interface IExtendedSplashscreenController
 /// </summary>
 public class MockExtendedSplashscreenController : IExtendedSplashscreenController
 {
-	public void Dismiss()
+	public async void Dismiss()
```
Comment (jeanplevesque) [inline src/app/ApplicationTemplate.Presentation/Framework/Startup/IExtendedSplashscreenController.cs:20]: ?

File: src/app/ApplicationTemplate.Shared.Views/Framework/ExtendedSplashscreenController.cs [hunk 3]
```diff
@@ -1,22 +1,23 @@
...
 
-	public ExtendedSplashscreenController(DispatcherQueue coreDispatcher)
+	public ExtendedSplashscreenController(DispatcherQueue dispatcherQueue)
 	{
-		_coreDispatcher = coreDispatcher;
+		_dispatcher = dispatcherQueue;
 	}
 
-	public void Dismiss()
+	public async void Dismiss()
```
Comment (jeanplevesque) [inline src/app/ApplicationTemplate.Shared.Views/Framework/ExtendedSplashscreenController.cs:18]: never use `async void`

Comment (jeanplevesque) [reply src/app/ApplicationTemplate.Shared.Views/Framework/ExtendedSplashscreenController.cs:18]: This seems unrelated to the packages clean-up

**Review by jeanplevesque — CHANGES_REQUESTED** [review]

File: src/app/ApplicationTemplate.Shared.Views/Framework/ExtendedSplashscreenController.cs (hunk 3 above)
Comment (GabrielChouinardLetourneau) [reply src/app/ApplicationTemplate.Shared.Views/Framework/ExtendedSplashscreenController.cs]: Those changes were pushed by mistake. I reverted and kept the changes related to this PR

File: src/app/ApplicationTemplate.Client/ApplicationTemplate.Client.csproj [hunk 4]
```diff
@@ -16,14 +16,13 @@
 		<PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="6.0.0" />
 		<PackageReference Include="Microsoft.Extensions.Http" Version="6.0.0" />
 		<PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="6.0.0" />
-		<PackageReference Include="Reactive.Annex.Uno.WinUI" Version="0.5.0-dev.44" />
+		<PackageReference Include="Reactive.Annex" Version="0.5.0-dev.46" />
 		<PackageReference Include="Refit" Version="6.3.2" />
 		<PackageReference Include="System.Text.Json" Version="6.0.6" />
 		<PackageReference Include="MallardMessageHandlers" Version="0.2.0-dev.21" />
+		<PackageReference Include="Nventive.Persistence.Reactive" Version="0.2.0-dev.28" />
```
Comment (carlh98) [inline src/app/ApplicationTemplate.Client/ApplicationTemplate.Client.csproj:23]: 0.3.0-dev.30 for all the nventive.persistence packages

**Pull request closed without merging**